src/javascript/app/base/header.js

```diff
@@ -728,6 +731,10 @@ const Header = (() => {
             low_risk_non_eu_accordion_header.style.display    = 'none';
             low_risk_eu_accordion_header.style.display        = 'none';
             low_risk_eu_container.style.display               = 'none';
+        } else if (Client.isLowRisk() && !has_mf_account) {
+            low_risk_non_eu_accordion_header.style.display    = 'none';
```

Comment:
So, it still will be in there, but with display none?

Reply:
yeah i mean. its not like we are stopping anyone from creating an account via api